custom/enikshay/integrations/nikshay/tests/test_repeaters.py

```diff
@@ -88,6 +90,9 @@ def _create_nikshay_registered_case(self):
         )
         self.create_case(nikshay_registered_case)
 
+    def update_case_with(self, case_id, case_attrs):
```

Comment:
added this method to avoid calling update_case with domain. It calls `update_case` internally. Just makes things better to read and at times in a single line.